### PR TITLE
02 css positionning

### PR DESCRIPTION
**Pull request by guillaume-boutin**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -1 +1,31 @@
 /* write your css here */
```
Comment (nyanofthemoon) [inline css/main.css]: @guillaume-boutin Is this from exercise 1?

File: index.html [hunk 2]
```diff
@@ -7,8 +7,29 @@
     <link rel="stylesheet" href="css/main.css">
```
Comment (nyanofthemoon) [inline index.html]: @guillaume-boutin Is this from exercise 1? Please remove as it is not used in your code.

**Comment by guillaume-boutin** [comment]
I put content in exercise02.html and ex02.css, and left index.html and main.css as it is. I will simply re-write index and main, and get rid of exercise02.html and ex02.css.

**Comment by nyanofthemoon** [comment]
@guillaume-boutin Please follow operational instructions for this workshop: "Each exercise goes to a different branch, branched from master"

**Comment by nyanofthemoon** [comment]
@guillaume-boutin Thank you- could you please rename the branch to something like using `git branch -m 02-css-positionning 01-something` so that I don't end up correcting the wrong exercise for the next branch?

**Comment by guillaume-boutin** [comment]
@nyanofthemoon 
01-navigation-menu
02-css-positionning
03-css-layouts
master
are the names of my branches. I'm not sure of what you are asking me.

**Comment by nyanofthemoon** [comment]
@guillaume-boutin This pull request is linked to branch "02-css-positionning" but contains material for exercise 1.

**Comment by nyanofthemoon** [comment]
@guillaume-boutin Oh wait, you moved the content around. Please ignore.

File: css/main.css [hunk 3]
```diff
@@ -1 +1,24 @@
...
+}
+.dialog-box__close {
+  border: 2px solid white;
+  background-color: tomato;
+  width: 24px;
+  height: 24px;
+  position: absolute;
+  top: 15px;
+  right: 15px;
+}
```
Comment (nyanofthemoon) [inline css/main.css]: @guillaume-boutin Just one missing one part of the puzzle and we can close the pull request :dancer:

Exerpt from exercise 1:
```Read more about box sizing on Paul Irish's blog and copy/paste his fix at the top of your main.css
Refesh your page to see that the width is now back to 100% total.
IMPORTANT: You should add this snippet of code to the top of your main.css for all future exercises! You can either do it every time, or commit it once to master and branch off of there. Your choice.```

**Comment by nyanofthemoon** [comment]
@guillaume-boutin Scrumptious! :+1:

**Pull request closed without merging**